production-marathon/marathon.env

```diff
@@ -0,0 +1,16 @@
+# The following properties MUST be configured before the Job Service can be deployed.
+
+export DOCKER_REGISTRY=NOT_SET_QA_index.docker.io/jobservice
```

Comment:
The suffix after NOT_SET_ should match the actual setting as far as I understand. For example, NOT_SET_DOCKER_REGISTRY